srcpkgs/duc/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'duc'
+pkgname=duc
+version=1.4.4
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="pango-devel cairo-devel ncurses-devel tokyocabinet-devel"
+short_desc="Library and suite of tools for inspecting disk usage"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="LGPL-3.0-only"
+homepage="https://duc.zevv.nl/"
+distfiles="https://github.com/zevv/"$pkgname"/releases/download/"$version"/"$pkgname"-"$version".tar.gz"
+checksum=f4e7483dbeca4e26b003548f9f850b84ce8859bba90da89c55a7a147636ba922
+
+post_extract() {
+	sed -i 's#<ncursesw/ncurses.h>#<ncurses.h>#' "$wrksrc"/src/duc/cmd-ui.c
+}
+
+post_install() {
+	vlicense COPYING
```

Comment:
GPL and LGPL licenses do not need to be installed.

Reply:
Removed for the template :)